packages/buildessential.rb

```diff
@@ -90,9 +90,6 @@ class Buildessential < Package
   # depends_on 'openjdk11'
   # depends_on 'openjdk17'
 
-  # Linter
-  depends_on 'linter'
-
```

Comment:
Might be good to replace linter with py3_pre_commit.

Reply:
`py3_pre_commit` is only really useful to chromebrew developers-- no need to install it for users.

Reply:
The entire essence of the `buildessential` package is for those who want to build from source and check packages.

Reply:
The `buildessential` packages are automatically installed if we build a package from source-- there are instances where a user needs to build a package from source but is not planning to contribute it to chromebrew (packages missing binaries, packages requiring local rebuilds for glibc reasons, etc)